### ISSUE TITLE
[Bug] : opacity percentages remove zeroes

### ISSUE THREAD

**Issue by jjwwiodyne** [cssnano/cssnano#962]
**Describe the bug**
`100%` is transformed to `1%` for `opacity` properties:

```
@-webkit-keyframes fade {
    0% { opacity: 100%; }
   40% { opacity: 0%; }
   60% { opacity: 0%; }
  100% { opacity: 100%; }
}
```

Becomes:

`@-webkit-keyframes fade{0%{opacity:1%}40%{opacity:0}60%{opacity:0}to{opacity:1%}}`

**To Reproduce**

1. Go to the cssnano playground: https://cssnano.co/playground
2. Add the CSS above.
3. Click Run.

**Expected behavior**
Expected:
`@-webkit-keyframes fade{0%{opacity:100%}40%{opacity:0}60%{opacity:0}to{opacity:100%}}`

**Comment by anikethsaha** [comment]
sorry for the delay
Can you try this with `cssnano@nightly` if the issue still exists ?

**Comment by bdoodo** [comment]
@anikethsaha Is there a way to set up cssnano@nightly on a parcel configuration? Using your minifier with Parcel.

**Comment by anikethsaha** [comment]
Not really familiar with parcel's config. But you would use nightly same as you use cssnano.

**Comment by alexbrault** [comment]
Bug title is slightly misleading; the code doesn't remove zeroes, it's just overly eager to clamp the opacity between 0 and 1 and doesn't take into account percentages. Since most non-zero percentages will be over 1, the clampOpacity function clamps it to 1, then adds back the percentage

**Comment by ludofischer** [comment]
Fixed in 5.0.0-rc.2
